src/Components/ProductCard/AvailabelColors.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Radio from "@mui/material/Radio";
+import RadioGroup from "@mui/material/RadioGroup";
+import FormControl from "@mui/material/FormControl";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import RadioButtonCheckedIcon from "@mui/icons-material/RadioButtonChecked";
+import RadioButtonUncheckedRoundedIcon from "@mui/icons-material/RadioButtonUncheckedRounded";
+
+const AvailabelColors = ({ availabelColors }) => {
+
+  return (
+    <div>
+      <FormControl>
+        <RadioGroup
+          row
+          aria-labelledby="demo-row-radio-buttons-group-label"
+          name="row-radio-buttons-group"
+        >
+          {availabelColors.map((colorItem, index) => {
```

Comment:
You are not using a unique key for the returned component. 
Please also don't use the index from the map method, [see why in the Pitfall section](https://react.dev/learn/rendering-lists#why-does-react-need-keys)